### PR TITLE
priv-1.10 merge

### PR DESCRIPTION
**Pull request by palmer-dabbelt**

### CODE REVIEW

**Comment by palmer-dabbelt** [comment]
I'm staging all the priv-1.10 branches for now, just so I don't forget about them.

File: fesvr/dtm.h [hunk 1]
```diff
@@ -20,14 +20,14 @@ class dtm_t : public htif_t
...
+    uint32_t data;
   };
 
   struct resp {
     uint32_t resp;
-    uint64_t data;
+    uint32_t data;
   };
-
+
```
Comment (colinschmidt) [inline fesvr/dtm.h]: white space

File: fesvr/dtm.h [hunk 2]
```diff
@@ -78,16 +87,18 @@ class dtm_t : public htif_t
 
   bool req_wait;
   bool resp_wait;
-  uint32_t dminfo;
+  //uint32_t ram_base;
```
Comment (colinschmidt) [inline fesvr/dtm.h]: remove commented code

File: fesvr/dtm.h [hunk 3]
```diff
@@ -78,16 +87,18 @@ class dtm_t : public htif_t
...
   uint32_t xlen;
 
   static const int max_idle_cycles = 10000;
 
-  size_t ram_base() { return 0x400; }
-  size_t rom_ret() { return 0x804; }
-  size_t ram_words() { return ((dminfo >> 10) & 63) + 1; }
+  size_t ram_words;
+  size_t data_words;
+
```
Comment (colinschmidt) [inline fesvr/dtm.h]: white sapce

**Pull request merged**